instrumentation/runtime-metrics/runtime-metrics-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetricsjava8/JmxRuntimeMetricsUtil.java

```diff
@@ -3,17 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.instrumentation.runtimemetrics;
+package io.opentelemetry.instrumentation.runtimemetricsjava8;
 
 import io.opentelemetry.api.OpenTelemetry;
 import io.opentelemetry.api.metrics.Meter;
 import io.opentelemetry.api.metrics.MeterBuilder;
 import io.opentelemetry.instrumentation.api.internal.EmbeddedInstrumentationProperties;
+import java.util.List;
 import javax.annotation.Nullable;
 
-class RuntimeMetricsUtil {
+class JmxRuntimeMetricsUtil {
 
-  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.runtime-metrics";
+  private static final String INSTRUMENTATION_NAME =
+      "io.opentelemetry.instrumentation.runtime-metrics";
```

Comment:
We usually name our instrumentations `io.opentelemetry.<name-version>`
```suggestion
      "io.opentelemetry.runtime-metrics-java8";
```

Reply:
updated